pkg/tasks/alertmanager.go

```diff
@@ -20,8 +20,11 @@ import (
 	"github.com/openshift/cluster-monitoring-operator/pkg/client"
 	"github.com/openshift/cluster-monitoring-operator/pkg/manifests"
 	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/types"
 )
 
+const MonitoringNamespace = "openshift-monitoring"
```

Comment:
is there a pkg where we keep all global constant?

Reply:
I realized we don't need to use const here since we can use `t.client.Namespace()` so I have removed the const